samcli/cli/main.py

```diff
@@ -58,6 +59,27 @@ def print_info(ctx, param, value):
     ctx.exit()
 
 
+def log_cmdline_info(func):
```

Comment:
Why not simply print `kwargs` as-is? Seems very complex for what's essentially a simulation of what the command could look like.

Reply:
Just want to make the format better.

Reply:
I'd argue it's overfitted to a very specific use case (and without docstring `log_cmdline_info` suggests it's generic). If `value` is a `list` or `dict` or `set` then the output will be weird.

Reply:
Thanks for mentioning this, the docstring was missed. It's added.